test/onnx/onnx_test_common.py

```diff
@@ -312,6 +312,24 @@ def run_ort(
     else:
         ort_model = onnx_model
 
+    # NOTE: Inline model before running in onnxruntime.
+    # This is a workaround since onnxruntime crashes or segfaults when loading model
+    # with nested functions.
+    # Ref: https://github.com/microsoft/onnxruntime/issues/15849
+    try:
+        import onnx.inliner
+    except ImportError:
+        warnings.warn("Cannot import onnx.inliner. Skip inlining model.")
+    else:
+        if isinstance(ort_model, bytes):
+            buffer = io.BytesIO(ort_model)
+        else:
```

Comment:
good catch, it seems redundant.

Reply:
no it is not redundant. I can add assert if you'd like but i feel it is dup checking since onnx_model is type annotated and this function is guarded by beartype.